ansible/roles/test/files/ptftests/sad_path.py

```diff
@@ -181,3 +209,66 @@ def sad_bgp_verify(self):
             self.log.append('BGP state down as expected on DUT')
         else:
             self.fails['dut'].add('BGP state not down on DUT')
+
+    def populate_lag_state(self):
+        if self.oper_type == 'neigh_lag_down':
+            self.neigh_lag_state = 'disabled'
+            self.lag_members_down = self.vm_dut_map[self.neigh_name]['dut_ports']
+        elif self.oper_type == 'dut_lag_down':
+            self.lag_members_down = self.vm_dut_map[self.neigh_name]['dut_ports']
+            self.neigh_lag_state = 'notconnect'
+
+    def change_dut_lag_state(self, is_up=True):
+        state = ['shutdown', 'startup']
+        self.log.append('Changing state of %s from DUT side to %s' % (self.vm_dut_map[self.neigh_name]['dut_portchannel'], state[is_up]))
+        stdout, stderr, return_code = self.cmd(['ssh', '-oStrictHostKeyChecking=no', self.dut_ssh, 'sudo config interface %s %s' % (state[is_up], self.vm_dut_map[self.neigh_name]['dut_portchannel'])])
```

Comment:
>'sudo config interface %s %s' % (state[is_up], self.vm_dut_map[self.neigh_name]['dut_portchannel']) [](start = 99, length = 99)

extract to another statement, don't make any one too much complex.

Reply:
done